unused/Battle_of_Gaixia.md

```diff
@@ -0,0 +1,18 @@
+ 
+The Battle of Gaixia took place in 202 BC and was an event that determined the course of Chinese history. It was fought over the entirety of China between Han forces led by Liu Bang and Chu forces led by Xiang Yu.
+At the time of this battle, the war between the Han and Chu had been going on for years. And it had not been going badly for the Chu and Xiang Yu. Despite his numerically smaller armies, he had been able to push back the Han. However, because Liu Bang's generals had come to help and threatened to encircle him, he was forced to withdraw to the Chu capital. The most significant of these generals was Han Xin.
+Han Xin's men were continuously attacking the retreating Chu army's supply lines and carrying out small ambushes. This exhausted the Chu and lowered their fighting ability. And the Han managed to capture Xiang Yu's own wife and bring her to the canyon of Gaixia. Even though Xiang Yu was well aware that this was a trap, he was so desperate to save his wife that after sending a large portion of his troops to defend the capital, he attempted to free his wife himself with the remaining 100,000 men.
+But once the Chu and Xiang Yu had entered the canyon, the Han struck. What happened was later described as an “ambush from ten sides”. In the end, even though Xiang Yu had managed to free his wife, he had lost almost his entire army and was now stuck in the canyon. At night, the Han played Chu folk songs, which was what ultimately broke the last Chu soldiers' morale. They, including Xiang Yu, thought their capital had already been taken and defeat was now unavoidable. Many of his soldiers deserted. And it got even worse for Xiang Yu: his wife, feeling responsible for the defeat, took her own life in front of Xiang Yu's eyes.
+And it was in this hopeless situation that Xiang Yu wrote down the following verses, known as the Song of Gaixia:
+ 
+“My strength plucked up the hills,
+My might shadowed the world,
+But the times were against me,
+And Zhui [his beloved horse] runs no more,
+When Zhui runs no more,
+What then can I do?
+Ah, Yu [his wife], my Yu,
+What will your fate be?”
+ 
+The next day, Xiang Yu was even able to break through and flee; however, he was pursued and eventually took his own life.
```

Comment:
```suggestion
The next day, Xiang Yu was even able to break through and flee; however, he was pursued and instead of facing captivity took his own life.
```

Reply:
"instead of facing captivity" - Although that might have actually been the case, the legend says he had the opportunity to get away but decided against it and to fight until death. It is also said that he, knowing that he had a high bounty, upon recognising a friend among the Han killed himself to do his friend a favour one last time.

Reply:
I changed it to "in shame for this devastating defeat", does that sound good?

Reply:
Yes.